src/api/v1/db/oracledb/onbase-dao.js

```diff
@@ -84,25 +99,17 @@ const patchOnBase = async (osuId, body) => {
   const connection = await getConnection();
   const { attributes } = body.data;
   try {
+    if (await personNotExist(connection, osuId)) {
+      throw createError(404, `${osuId} - No person exists`);
```

Comment:
This string is used twice now. Should we save it some where so it can be referenced instead?

Reply:
Good catch, will fix.